TransactionService/docker_context/handlers/BaseLogicHandler.py

```diff
@@ -0,0 +1,117 @@
+import pickle
+from datetime import datetime
+
+class BaseLogicHandler:

```

Comment:
just thinking we rename this to TransactionHandler along with BaseLogic --> TransactionLogic

Reply:
Rename complete.